src/e2e-test/features/mssql/DesignTimeValidation.feature

```diff
@@ -0,0 +1,138 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Mssql
+Feature: Mssql - Verify Mssql source plugin design time validation scenarios
+
+  Scenario: To verify validation message when user provides invalid Host
+    Given Open DataFusion Project with replication to configure pipeline
+    When Enter input plugin property: "name" with pipelineName
+    And Click on the Next button in Replication mode "Next"
+    And Select Source plugin: "Microsoft SQLServer" from the replication plugins list
+    Then Replace input plugin property: "host" with value: "mssqlInvalidHost" for Credentials and Authorization related fields
+    Then Replace input plugin property: "port" with value: "mssqlPort" for Credentials and Authorization related fields
+    Then Select dropdown plugin property: "select-jdbcPluginName" with option value: "mssqlDriverName"
+    Then Replace input plugin property: "database" with value: "mssqlDatabaseName"
+    Then Replace input plugin property: "user" with value: "mssqlUsername" for Credentials and Authorization related fields
+    Then Replace input plugin property: "password" with value: "mssqlPassword" for Credentials and Authorization related fields
+    And Click on the Next button in Replication mode "Next"
```

Comment:
I guess you can hard code the data-cy or text for this step  in the locator. I don't think the locator will be reusable for any other use case.

Reply:
I have added this step in framework, do I need to create and use it from here?